shared/actions/chat/creators.js

```diff
@@ -113,10 +113,10 @@ const loadAttachmentPreviewTransformer = ({
   type,
 })
 
-function exitSearch(): Constants.ExitSearch {
+function exitSearch(forSelectConversation: boolean): Constants.ExitSearch {
```

Comment:
I couldn't think of a better name/way to signal exitSearch what I want. Suggestions welcome!